build_locally.sh

```diff
@@ -14,7 +14,7 @@ sphinx-build -W -b html . native_build
 rosdoc_lite -o build .
 
 # Validate if the HTML is valid (i.e. no missing links, images, etc.)
-htmlproofer ./build --only-4xx --check-html --file-ignore ./build/html/genindex.html,./build/html/search.html,./build/html/index-msg.html --alt-ignore '/.*/' --url-ignore '#'
+htmlproofer ./build --only-4xx --check-html --file-ignore --http_status_ignore 429 ./build/html/genindex.html,./build/html/search.html,./build/html/index-msg.html --alt-ignore '/.*/' --url-ignore '#'
```

Comment:
It should be `--https-status-ignore` and it is in the wrong spot. `--file-ignore` accepts a list of files to ignore, so now it will ignore the files `--http_status_ignore` and `429` :upside_down_face:. So, in order to fix it you should move the argument to a correct position.
```suggestion
htmlproofer ./build --only-4xx --http-status-ignore 429 --check-html --file-ignore ./build/html/genindex.html,./build/html/search.html,./build/html/index-msg.html --alt-ignore '/.*/' --url-ignore '#'
```